### PR TITLE
Updated known issues for 1.4.6 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated known issues for 1.4.6

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,7 +31,19 @@ The [1.5 release](docs/1_5_Upgrade.md) is scheduled for early July.
...
+could cause Besu to crash. [\#986](https://github.com/hyperledger/besu/pull/986)
+- Fixed issue where networks using onchain permissioning could stall when the bootnodes were not validators. [\#969](https://github.com/hyperledger/besu/pull/969)
+
+### Known Issues 
+
+Known issues are open issues categorized as [Very High or High impact](https://wiki.hyperledger.org/display/BESU/Defect+Prioritisation+Policy).
+
+#### Previously identified known issues
+ 
+- [Eth/65 not backwards compatible](KNOWN_ISSUES.md#eth65-not-backwards-compatible)
```
Comment (shemnon) [inline CHANGELOG.md:44]: The backwards incompatibility part of ETH/65 is fixed.

Comment (MadelineMurray) [reply CHANGELOG.md:44]: Updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -31,7 +31,19 @@ The [1.5 release](docs/1_5_Upgrade.md) is scheduled for early July.
 
 ### Bug Fixes
 
-- Fixed case where networks using onchain permissioning could stall when the bootnodes were not validators. [\#969](https://github.com/hyperledger/besu/pull/969)
+- Added timeout to queries.  Fixes issue where `eth_getLogs` queries that were too large or too broad 
+could cause Besu to crash. [\#986](https://github.com/hyperledger/besu/pull/986)
```
Comment (shemnon) [inline CHANGELOG.md:35]: Besu can still crash/hang, that's the deeper fix we mentioned.
```suggestion
- Added timeout to queries.  Fixes issue where `eth_getLogs` queries that were too large or too broad 
never return. [\#986](https://github.com/hyperledger/besu/pull/986)
```

Comment (MadelineMurray) [reply CHANGELOG.md]: Fixed and added the known issue back.  Thanks for catching my miss.

**Comment by sambacha** [comment]
Fast Sync has always worked for us on AWS? curious, just literally spun up an instance and tried using the 1.4.6-rc1 and was able to fast sync on AWS

**Pull request merged**